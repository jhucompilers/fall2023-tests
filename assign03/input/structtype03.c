struct Point {
  int x, y;
};

int main(void) {
  struct Point points[10];
  int i;

  for (i = 0; i < 10; i = i + 1) {
    points[i].x = i;
    points[i].y = i;
  }

  return 0;
}
