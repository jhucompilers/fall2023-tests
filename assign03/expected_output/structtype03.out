0|struct Point|type|struct Point {}
1|x|variable|int
1|y|variable|int
0|main|function|function () returning int
2|points|variable|array of 10 x struct Point {int, int}
2|i|variable|int
